spec/support/rails_support.rb

```diff
@@ -1,6 +1,6 @@
 module RailsSupport
   def create_rails_app
-    in_current_dir { FileUtils.rm_rf 'testapp' }
+    cd('.') { FileUtils.rm_rf 'testapp' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.